extensions/tmtv/src/index.tsx

```diff
@@ -26,6 +27,16 @@ const tmtvExtension = {
       extensionManager,
     });
   },
+  getUtilityModule() {
```

Comment:
We are moving away from the utility module. You can export it in this extension and have other extensions depend on it and import it there.